why-python.qmd

```diff
@@ -112,10 +139,17 @@ What decision was made, use the form "We decided on CHOICE because of
 REASONS."
 :::
 
+We have decided to use Python as the main development language for Seedcase because it is a powerful, flexible and easy-to-use platform for building a data management web application. Drawing on the skills of the core Seedcase team, Python offers the best balance of capabilities and ease of development out of the options considered. It is, moreover, a language likely to be familiar to technologically-oriented Seedcase users and prospective contributors.
+
+While Java and C++ offer more scope for context-specific performance optimisation, we are unlikely to need this level of control for Seedcase, because the number of concurrent user requests is expected to be low, resource-intensive backend tasks are expected to run without much competition, and we don't expect to do heavy-duty, real-time data analysis or image processing. Therefore, for our use case, the advantages of these languages don't justify their steeper learning curve and added complexity.
```

Comment:
Here I'm thinking that Seedcase won't serve thousands of user requests at once, won't run very many high-workload scheduled jobs in the background and won't do super complex calculations on data. But that's just my understanding so far, so let me know if that's not the case!

Reply:
Honestly sounds about right, but maybe @lwjohnst86 would like to comment as well.

Reply:
@martonvago you got it!!